### ISSUE TITLE
SA1004 should report a warning for indented top-level elements

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#1238]
Currently SA1004 does not report a warning for the following code:

```csharp
/// <summary>
/// The summary text.
/// </summary>
///   <param name="x">The document root.</param>
///    <param name="y">The Xml header token.</param>
private void Method1(int x, int y)
{
}
```

According to the documentation for this rule, it should report a warning here. The exclusion for indented lines should only apply to *content*, such as the following:

```csharp
/// <summary>
///     The summary text.
/// </summary>
/// <param name="x">The document root.</param>
/// <param name="y">The Xml header token.</param>
private void Method1(int x, int y)
{
}
```
